acme_diags/derivations/acme.py

```diff
@@ -361,12 +375,16 @@ def cosp_bin_sum(cld, prs_low0, prs_high0, tau_low0, tau_high0):  # noqa
             )  # MODIS obs
 
     if cld.id == "CLD_MISR":  # MISR model
+        if max(prs) > 1000:  # COSP v2 cosp_htmisr in units m instead of km as in v1
+            cld = cld[
+                1:, :, :, :
+            ]  # COSP v2 cosp_htmisr[0] equals to 0 instead of -99 as in v1, therefore cld needs to be masked manually
         cld_bin = cld(cosp_htmisr=(prs_low, prs_high), cosp_tau=(tau_low, tau_high))
-        if prs_low == 7:
+        if prs_low in [7, 7000]:
             prs_lim = "high cloud fraction"
-        if prs_high == 7 and prs_low == 3:
+        if prs_high in [7, 7000] and prs_low in [3, 3000]:
```

Comment:
Same question related to swapping the bounds around

Reply:
same as above.